src/workloads/scale/TimeSeriesSortScale.yml

```diff
@@ -2,531 +2,83 @@ SchemaVersion: 2018-07-01
 Owner: "@mongodb/query"
 Description: |
     This test exercises the behavior of the time series bounded sorter as the number of documents
-    in the collection increases.  We test the sort from 1 million documents to 15 million. Each
-    document has a random meta value ranging from 0 to 1000. Since this file is meant to test how
-    the bounded sorter scales with collection document count, we do not increase the number of
-    series during each insertion phase. The bounded sorter is expected to scale linearly, while a
-    normal blocking sort should scale worse.
+    in the collection increases.  The collection has 10 million documents, and each document has a
+    random meta value ranging from 0 to 1000. The bounded sorter is expected to scale linearly,
+    while a normal blocking sort should scale worse.
```

Comment:
```suggestion
    random meta value ranging from 0 to 1000.
```

Reply:
I think this last sentence doesn't apply any more, because we're not testing multiple collection sizes (scaling) and we're not comparing against the naive plan.